### PR TITLE
Nested classes to fix package-private issue

### PR DESCRIPTION
**Pull request by bhjelstrom**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,14 +18,14 @@ getLatestResults() parses the latest Limelight JSON dump into a LimelightResults
 Takes up to 2.5ms on RoboRIO 1.0. Parsing latency is logged in results.targetingresults.latency_jsonParse and may be added to latency_pipeline and latency_capture.
 
 ```
-LimelightResults llresults = LimelightHelpers.getLatestResults("");
+LimelightHelpers.LimelightResults llresults = LimelightHelpers.getLatestResults("");
```
Comment (agasser) [inline README.md:21]: Maybe consider using `var`, it makes it look more simple.

```java
var llresults = LimelightHelpers.getLatestResults("");
```

**Review by agasser — COMMENTED** [review]
This still doesn't work. Pretty much everything has to be `public`. Package private (default) visibility applies to nested classes and member variables as well. Just nesting the classes doesn't increase their visibility. Nesting is just a workaround to cram everything into a single file.

**Comment by bhjelstrom** [comment]
Did you see the third commit which changes the visibility of the nested classes? All members need to be explicitly marked public as well?

**Comment by agasser** [comment]
Yes, all members have to be public too or they cannot be used outside of `frc.robot`.

**Pull request merged**